README.rst

```diff
@@ -1,96 +1,90 @@
 edx-sysadmin
 =============================
 
-|pypi-badge| |travis-badge| |codecov-badge| |doc-badge| |pyversions-badge|
-|license-badge|
+This is a django app plugin extracted from `edx-platform <https://github.com/edx/edx-platform>`_ which enables certian users to perform some specific operations in Open edX environment (which are described under ``Features`` section below).
+Earlier, ``Sysadmin Dashboard`` was a part of ``edx-platform``, however starting from `lilac release <https://github.com/edx/edx-platform/tree/open-release/lilac.master>`_ of Open edX the sysadmin panel has been removed
+and transitioned to as separate plugin.
 
-The ``README.rst`` file should start with a brief description of the repository,
-which sets it in the context of other repositories under the ``edx``
-organization. It should make clear where this fits in to the overall edX
-codebase.
+NOTE:
+It is recommended that you use edx-sysadmin plugin with Open edX's `lilac <https://github.com/edx/edx-platform/tree/open-release/lilac.master>`_ release and successors.
+If you wish to use the ``Sysadmin Dashboard`` with Open edX releases before ``lilac`` you should just enable ``ENABLE_SYSADMIN_DASHBOARD`` feature flag in environment files (e.g ``lms.yml`` or ``lms.env.json``) to access sysadmin dashboard features.
```

Comment:
@HamzaIbnFarooq I don't think we are not relying on `ENABLE_SYSADMIN_DASHBOARD` feature flag. we just need to install plugin and sys admin should be available.

Reply:
It says `If you wish to use the Sysadmin Dashboard with Open edX releases before lilac you`. Only meant for old releases.

Reply:
I have updated the sentence now as it might had made a wrong sense before, kindly have a look now.